libs/sdk-ui-kit/styles/scss/shareDialog.scss

```diff
@@ -283,37 +285,73 @@
         }
     }
 
-    &:hover {
+    &:hover,
+    &.is-active {
         background: variables.$is-focused-background;
 
         .gd-grantee-item-icon-right {
             display: block;
         }
     }
-}
 
-.gd-granular-permissions-dropdown {
-    display: flex;
-    align-items: center;
-    margin-right: 10px;
+    .gd-grantee-granular-permission {
+        display: flex;
+        align-items: center;
+        margin-right: 10px;
+    }
+
+    .gd-granular-permission-button {
+        display: flex;
+        justify-content: flex-end;
+        align-items: center;
+        width: $permissions-dialog-width;
+        height: 27px;
+        padding-right: 8px;
+        border: 1px solid transparent;
+        font-family: variables.$gd-font-primary;
+
+        &.disabled {
+            color: var(--gd-palette-complementary-5, #b0beca);
+            background: rgba(252, 252, 253, 0.3);
+        }
+
+        &:hover {
+            cursor: pointer;
+            background-color: var(--gd-palette-complementary-0, #fff);
+            border: 1px solid var(--gd-palette-complementary-4, rgba(31, 52, 73, 0.2));
+            border-radius: 3px;
+        }
+
+        &.is-active {
+            border-radius: 3px;
+            box-shadow: inset 0 1px 0 0 var(--gd-shadow-color, rgba(177, 193, 209, 0.65));
+            border-color: var(
+                --gd-palette-complementary-5,
+                var(--gd-palette-complementary-4, rgba(31, 52, 73, 0.2))
+            );
+            background-color: var(--gd-palette-complementary-2, #ecf0f5);
+        }
+
+        .gd-granular-permission-button-title {
+            margin-right: 6px;
+            font-size: 14px;
+            font-weight: normal;
+            line-height: 20px;
+            color: var(--gd-palette-complementary-6, #687581);
+        }
+
+        &::after {
+            margin-top: 2px;
+        }
+    }
 }
 
 .gd-granular-permissions-remove {
     border-top: 1px solid variables.$gd-border-color;
 }
 
-.gd-granular-permission-button {
```

Comment:
this ccs change could be dangerous too.

Reply:
This `granular permission button` was introduced in previous PRs, but it is still part of this whole new granular flow, so nobody is using it yet. Should be ok.